production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json

```diff
@@ -4799,180 +5784,153 @@
+                     "exemplars": {
+                        "color": "rgba(255,0,255,0.7)"
+                     },
+                     "filterValues": {
+                        "le": 1.0000000000000001e-09
+                     },
                      "legend": {
-                        "calcs": [ ],
-                        "displayMode": "list",
-                        "placement": "bottom",
-                        "showLegend": true
+                        "show": true
+                     },
+                     "rowsFrame": {
+                        "layout": "auto"
                      },
                      "tooltip": {
-                        "maxHeight": 600,
                         "mode": "single",
-                        "sort": "none"
+                        "showColorScale": false,
+                        "yHistogram": false
+                     },
+                     "yAxis": {
+                        "axisPlacement": "left",
+                        "reverse": false
                      }
                   },
                   "panels": [ ],
+                  "pluginVersion": "11.4.0-77663",
                   "targets": [
                      {
                         "datasource": {
                            "type": "prometheus",
                            "uid": "${datasource}"
                         },
                         "editorMode": "code",
-                        "expr": "histogram_quantile(0.99, sum(rate(loki_bloom_store_download_queue_enqueue_time_seconds_bucket{cluster=\"$cluster\", namespace=\"$namespace\", container=\"bloom-gateway\"}[$__rate_interval])) by (le))",
-                        "hide": false,
-                        "legendFormat": "Enqueue latency",
-                        "range": true,
-                        "refId": "A"
-                     },
-                     {
-                        "datasource": {
-                           "type": "prometheus",
-                           "uid": "${datasource}"
-                        },
-                        "editorMode": "code",
-                        "expr": "histogram_quantile(0.99, sum(rate(loki_bloom_store_download_queue_size_bucket{cluster=\"$cluster\", namespace=\"$namespace\", container=\"bloom-gateway\"}[$__rate_interval])) by (le))",
+                        "expr": "increase(loki_bloom_gateway_tasks_dequeued_bucket{cluster=\"$cluster\", namespace=\"$namespace\", container=\"bloom-gateway\"}[$__rate_interval])",
+                        "format": "heatmap",
                         "hide": false,
-                        "legendFormat": "Size",
+                        "instant": false,
+                        "legendFormat": "__auto",
                         "range": true,
                         "refId": "B"
                      }
                   ],
-                  "title": "Block download queue",
-                  "type": "timeseries"
+                  "title": "Dequeue count",
```

Comment:
This panel is blank for me even though "Dequeue duration" isn't blank. Bug?

Reply:
Interesting, the metric definitely exists https://github.com/grafana/loki/pull/14413